examples/calculator/lib/calc_expression.dart

```diff
@@ -0,0 +1,329 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// A token that composes an expression. There are several kinds of tokens
+/// that represent arithmetic operation symbols, numbers and pieces of numbers.
+/// We need to represent pieces of numbers because the user may have only
+/// entered a partial expression so far.
+class ExpressionToken {
+  String stringRep;
+
+  ExpressionToken(this.stringRep);
+
+  String toString() {
+    return stringRep;
+  }
```

Comment:
We'd usually write this using the `=>` shorthand:

`String toString() => stringRep`